flink-core/src/main/java/org/apache/flink/api/common/io/DelimitedInputFormat.java

```diff
@@ -84,12 +84,11 @@
 	 */
 	private static int MAX_SAMPLE_LEN;
 
-	static { loadGlobalConfigParams(); }
-	
-	protected static void loadGlobalConfigParams() {
-		int maxSamples = GlobalConfiguration.getInteger(ConfigConstants.DELIMITED_FORMAT_MAX_LINE_SAMPLES_KEY,
+
+	protected static void loadConfigParameters(Configuration parameters) {
```

Comment:
I think this is a public API change. The class is annotated with `@Public` and the method is `protected`, which means that some users might have extended this input format and rely on this method. What do you think?